Source/Core/Constants+LocalizationKeys.swift

```diff
@@ -4,6 +4,10 @@ extension Constants {
 
         struct BillingForm {
             
+            struct Cell {
+                static let isOptional = "isOptional".localized
```

Comment:
Is this key already present? I can't see it on my version anywhere else, so I think its new?